e2e-tests/smart-update/run

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+IMAGE_PXC=percona/percona-xtradb-cluster-operator:1.4.0-pxc8.0
```

Comment:
I think we need to have the possibility to set  IMAGE_PXC and IMAGE_TO_UPDATE via env variables.

Reply:
agree

Reply:
done